.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: dart
 
 dart:
-  - dev
+  - preview/raw/2.10.0-0.2-dev
```

Comment:
weird to put this in master...

Reply:
I will get switched back to `dev` eventually (once there is a 2.10 dev)

Reply:
> Are we deciding we're okay breaking travis in the short term?

Right, but only very temporarily (unless something goes wrong)